### PR TITLE
Review J letter icons

### PR DESCRIPTION
**Pull request by jwr1**
**Issue:** contributes to #9072

* Jameson - Update SVG
* Jeep - Update source and guidelines
* JFrog - Update hex
* JR Group - Update hex
* JUnit5 - Fix source link to use github.com, not githubusercontent.com

_Icon removals will be in separate PRs_

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6703,7 +6702,7 @@
         },
         {
             "title": "JFrog",
-            "hex": "41BF47",
+            "hex": "40BE46",
             "source": "https://jfrog.com/brand-guidelines/",
             "guidelines": "https://jfrog.com/brand-guidelines/"
```
Comment (adamrusted) [inline _data/simple-icons.json:6707]:
```suggestion
            "source": "https://jfrog.com/brand-guidelines",
            "guidelines": "https://jfrog.com/brand-guidelines"
```

File: icons/jameson.svg
Comment (adamrusted) [inline icons/jameson.svg:1]: We still need to keep the outline, as per their website homepage. I'm seeing differences between our version in black, and their website in cream though.
image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/jameson.svg
Comment (jwr1) [reply icons/jameson.svg:1]: You're right, I only saw this version without the outline (before entering your id) and assumed they changed the logo:
image

Comment (jwr1) [reply icons/jameson.svg:1]: Also, the simpleicons version doesn't have the trademark on it (™), should I add that too, since it's in the original:

image

Comment (adamrusted) [reply icons/jameson.svg:1]: Nope, we're clear to remove ™️ symbols, but should always keep ®️ icons unless they're used without elsewhere!

**Comment by jwr1** [comment]
Actually, don't merge yet. I just noticed the new jeep source doesn't match our current svg.

**Comment by jwr1** [comment]
It should be good now :+1:.

Also, what was the reason for my other pull requests being closed (#9639, #9642, and #9640)? There wasn't any real explanation besides:

> We never remove icons in this way. Closing.

and

> Please stop sending these kinds of PRs. Closing.

I can understand for Jenkins X, since they have 4.4k stars, it's already close to the necessary rank, and for JPEG, it's just a popular format, but Jabber has a rank of 2,777,993 and I've never even heard of it before.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -6643,8 +6643,7 @@
         {
             "title": "Jeep",
             "hex": "000000",
-            "source": "https://www.fcaci.com/x/JEEPv15",
-            "guidelines": "https://www.fcaci.com/x/JEEPv15"
+            "source": "https://en.wikipedia.org/wiki/File:Jeep_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:6646]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Jeep_logo.svg"
```

File: icons/jeep.svg
Comment (adamrusted) [inline icons/jeep.svg:1]: Seeing a slight difference when comparing to source, could you have another look please @jwr1?
Might just be a precision thing...
image

File: icons/jameson.svg
Comment (adamrusted) [inline icons/jameson.svg:1]: SVG is spot on for this one, thanks @jwr1! 🎉

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/jeep.svg
Comment (jwr1) [reply icons/jeep.svg:1]: You were right; it needed a precision of 4 to work. It should be good now.

**Review by adamrusted — APPROVED** [review]
LGTM, thanks @jwr1 🎉

**Pull request merged**